src/utils/osx/OSXUtils.mm

```diff
@@ -1,5 +1,6 @@
 #include "OSXUtils.h"
 #include "QsLog.h"
+#include "ui/KonvergoWindow.h"
```

Comment:
We can more likely just use the QWindow class include here rather than using our specific window, would be more generic. ie `#include <QWindow>`